src/android/Capture.java

```diff
@@ -377,20 +381,54 @@ else if (resultCode == Activity.RESULT_CANCELED) {
         }
     }
 
-
     public void onAudioActivityResult(Request req, Intent intent) {
-        // Get the uri of the audio clip
-        Uri data = intent.getData();
-        if (data == null) {
-            pendingRequests.resolveWithFailure(req, createErrorObject(CAPTURE_NO_MEDIA_FILES, "Error: data is null"));
-            return;
+        Uri srcContentUri = intent.getData();
```

Comment:
Keep "pattern" between audio, image an video: comment, `Uri data = intent.getData()`, and check on null with resolveWithFailure?

Reply:
I updated the PR to perform the null check.
I kept the variable name as `srcContentUri` so we know exactly what the data contains.

Reply:
I agree with the variable name `srcContentUri` here as a way to self-document what the data is, or at least what was the expectation at the time of writing the code.

Reply:
Ok for the variable name 👍